src/l500/l500-color.cpp

```diff
@@ -382,6 +389,9 @@ namespace librealsense
                 LOG_DEBUG( "Closing color sensor..." );
                 synthetic_sensor::close();
             }
+
+            restore_pre_calibration_controls_if_needed();
```

Comment:
Not under if( _autocal )?

Reply:
The init function is under _autocal, I wanted to condition it as less as possible.
This way all of the logic flow can work , the vector will be empty.

+ It is for L515 device which will have auto cal.. it is not for a generic sensor flow.